src/modes.cpp

```diff
@@ -29,7 +35,7 @@ void aste::runReplayMode(const aste::ExecutionContext &context, const std::strin
     for (const auto dataname : asteInterface.writeVectorNames) {
       const int dataID = preciceInterface.getDataID(dataname, asteInterface.meshID);
       asteInterface.mesh.meshdata.push_back(aste::MeshData(aste::datatype::WRITE, dim, dataname, dataID));
-#ifdef ASTE_NN_GRADIENT_MAPPING
+#ifdef ASTE_NN_GRADIENT_MAPPING_AND_TETRA
```

Comment:
We can use the macro here directly, no need to add an additional variable.

Reply:
I see, it seems in first try I have a problem with spaces and cannot apply this, now solved and fixed.

Reply:
```suggestion
#if PRECICE_VERSION_GREATER_EQUAL(2, 5, 0)
```
This is what I meant by directly using the macro

Reply:
I changed and lost in VSCode somehow the commit send as comment :)